esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -695,6 +695,31 @@ bool CWsESDLConfigEx::onPublishESDLBinding(IEspContext &context, IEspPublishESDL
     return true;
 }
 
+void setContainerizedEspBindingNameByPort(StringBuffer& espPort, StringBuffer& espBindingName)
+{
+    Owned<IPropertyTree> globalConfig = getGlobalConfigSP();
+    if (!globalConfig)
+        throw makeStringExceptionV(-1, "Can't use global config to find any esp binding for port %s", espPort.str());
```

Comment:
getGlobalConfigSP() already throws an exception if globalconfig is not available.  Also the methods that end in SP can be used inline, so since you are only using globalConfig in one place (that I see) you don't need to declare the globalConfig varible.

Reply:
done